research/integrations/fedora-ci/index.md

```diff
@@ -0,0 +1,338 @@
+might need some tweaking as Packit has specific worfklow.
+
+### Koji scratch build
+
+We are already submitting scratch builds for merged PRs, if configured. This
+would just extend the functionality to the non-merged PRs (with need to provide
+the SRPM).
+
+### RPM linting, inspect and installability test
+
+There are premade tmt plans that we can run.
+
+:::note Question
+
+Should these be run in parallel or sequentially and fail on first?
+
+:::
+
+#### Installability test
+
+Installability test run by Fedora CI triggers Testing Farm and uses the
+following tmt plan: https://github.com/fedora-ci/installability-pipeline
+
+Installability test run by Zuul CI is done in the Zuul itself.
+
+### Testing Farm
+
+Different form of request than for upstream jobs.
+
+```json
+{
+  "id": "2fdee1a0-0746-45ad-ad0c-d3ec8833630b",
+  "user_id": "c4af7afe-b95e-4cf1-b989-9c458f0eecf0",
+  "token_id": "c4af7afe-b95e-4cf1-b989-9c458f0eecf0",
+  "test": {
+    "fmf": {
+      "url": "https://src.fedoraproject.org/forks/packit-stg/rpms/packit",
+      "ref": "0d4dfcfcc2a708f06c65b80404cc022e7bf6bead",
+      "merge_sha": null,
+      "path": ".",
+      "name": null,
+      "settings": null,
+      "plan_filter": null,
+      "test_name": null,
+      "test_filter": null
+    },
+    "script": null,
+    "sti": null
+  },
+  "state": "complete",
+  "environments_requested": [
+    {
+      "arch": "x86_64",
+      "os": {
+        "compose": "CentOS-Stream-9"
+      },
+      "pool": null,
+      "variables": {
+        "KOJI_TASK_ID": "123014869"
+      },
+      "secrets": null,
+      "artifacts": [
+        {
+          "id": "123014869",
+          "type": "fedora-koji-build",
+          "packages": null,
+          "install": true,
+          "order": 50
+        }
+      ],
+      "settings": null,
+      "tmt": {
+        "context": {
+          "arch": "x86_64",
+          "distro": "centos-stream-9",
+          "initiator": "fedora-ci",
+          "trigger": "commit"
+        },
+        "environment": null
+      },
+      "hardware": null,
+      "kickstart": null
+    }
+  ],
+  "notes": [
+    {
+      "level": "info",
+      "message": "tf-tmt/dispatch-1725615078-80d05f05"
+    }
+  ],
+  "result": {
+    "summary": null,
+    "overall": "passed",
+    "xunit": "…",
+    "xunit_url": "https://artifacts.dev.testing-farm.io/2fdee1a0-0746-45ad-ad0c-d3ec8833630b/results.xml"
+  },
+  "run": {
+    "console": null,
+    "stages": null,
+    "artifacts": "https://artifacts.dev.testing-farm.io/2fdee1a0-0746-45ad-ad0c-d3ec8833630b"
+  },
+  "settings": null,
+  "user": {
+    "webpage": null
+  },
+  "queued_time": 6.892783,
+  "run_time": 1207.608357,
+  "created": "2024-09-06T09:31:17.471815",
+  "updated": "2024-09-06T09:31:17.471844"
+}
+```
+
+Most notable differences:
+
+- RPMs are provided from Koji scratch build (following Fedora CI, which is
+  easier than self-hosting)
```

Comment:
Do you want to reuse the FedraCI-triggered Koji scratch build or trigger our own to be independent?

Reply:
:thinking: If Packit is to replace Fedora CI, there is no _Fedora CI-triggered Koji scratch build_ :P

Reply:
+ maybe also worth considering [this comment](https://github.com/packit/packit/issues/2414#issuecomment-2381490446)